### PR TITLE
Modified template files

### PR DESCRIPTION
**Pull request by akash2144**
Signed-off-by: Akash Dudhane <akash.a.dudhane@seagate.com>

## Problem Statement
<pre>
  <code>
  Story Ref (if any):
    Modify template files
  </code>
</pre>
## Unit testing on RPM done
<pre>
  <code>
  Yes
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Modify template file
  </code>
</pre>
## Solution
<pre>
  <code>
    Update template files : Added keys which are present in current provisioner conf store. 
  </code>
</pre>
## Unit Test Cases
<pre>
  <code>
    Confirmed that all mini-provisioning command works fine with updated template files.
  </code>
</pre>

### CODE REVIEW

**Comment by cortx-admin** [comment]
Can one of the admins verify this patch?

File: conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node [hunk 1]
```diff
@@ -19,5 +19,8 @@ server_node:
       management:
         interfaces: TMPL_INTERFACES
         netmask: TMPL_NETMASK
+    site_id: TMPL_SITE_ID
+    rack_id: TMPL_RACK_ID
+    storageset_id: TMPL_STORAGESET_ID
```
Comment (ajaysrivas) [inline conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node:24]: This should be storage_set_id. Can you please verify with conf store spec.

Comment (indrajitzagade) [reply conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node:24]: I see it as storage_set_id in the old provisioner conf store file that I have.

**Review by indrajitzagade — CHANGES_REQUESTED** [review]
Please fix review comment.

File: conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node (hunk 1 above)
Comment (akash2144) [reply conf/mini_provisioner/v2/ha.cleanup.tmpl.1-node]: "storageset_id" changed to "storage_set_id".

**Review by indrajitzagade — APPROVED** [review]

**Review by ajaysrivas — APPROVED** [review]

**Pull request merged**